app/assets/stylesheets/styles/_presets.css.scss

```diff
@@ -28,9 +28,14 @@
   display: inline-block;
 }
 
-@mixin button-box-shadow(){
-  -moz-box-shadow: 0px 2px 0px 0px rgba(0,0,0,0.08), 0px 4px 10px 0px rgba(0,0,0,0.10), inset 0px 1px 0px 1px rgba(255,255,255,0.1);
-  box-shadow: 0px 2px 0px 0px rgba(0,0,0,0.08), 0px 4px 10px 0px rgba(0,0,0,0.10), inset 0px 1px 0px 1px rgba(255,255,255,0.1);
+@mixin manual-font-awesome($glyph) {
```

Comment:
Is this okay? I started moving things towards a feel more similar to the Treehouse and in the end  reduced the shadows to the point where the mixin didn't make sense. How do you think the buttons look?